lib/route.js

```diff
@@ -414,7 +413,7 @@ function buildRouting (options) {
     if (context.logSerializers) {
       loggerOpts.serializers = context.logSerializers
     }
-    const childLogger = createChildLogger(context.server, logger, req, id, childLoggerOpts, loggerOpts)
+    const childLogger = createChildLogger(context.server, logger, req, id, context, loggerOpts)
```

Comment:
Maybe we can pass the whole context instead of the server and later the whole context? 
```suggestion
    const childLogger = createChildLogger(context, logger, req, id, loggerOpts)
```

Reply:
Do you mean behaviour over the `createChildLogger` function?